docs/sparkr.md

```diff
@@ -158,20 +152,19 @@ write.df(people, path="people.parquet", source="parquet", mode="overwrite")
 
 ### From Hive tables
 
-You can also create SparkR DataFrames from Hive tables. To do this we will need to create a HiveContext which can access tables in the Hive MetaStore. Note that Spark should have been built with [Hive support](building-spark.html#building-with-hive-and-jdbc-support) and more details on the difference between SQLContext and HiveContext can be found in the [SQL programming guide](sql-programming-guide.html#starting-point-sqlcontext).
+You can also create SparkDataFrames from Hive tables. To do this we will need to create a SparkSession with Hive support which can access tables in the Hive MetaStore. Note that Spark should have been built with [Hive support](building-spark.html#building-with-hive-and-jdbc-support) and more details can be found in the [SQL programming guide](sql-programming-guide.html#starting-point-sqlcontext). In SparkR, by default it will attempt to create a SparkSession with Hive support enabled (`enableHiveSupport = TRUE`).
```

Comment:
This will eventually be broken in the following PR for the SQL Programming  Guide.

https://github.com/apache/spark/pull/13592/files#diff-d8aa7a37d17a1227cba38c99f9f22511R55

- `#starting-point-sqlcontext` -> `#starting-point-sparksession`